src/main/java/org/jenkinsci/modules/windows_slave_installer/WindowsSlaveInstaller.java

```diff
@@ -72,35 +66,7 @@ public Localizable getConfirmationText() {
      * If it fails in a way that indicates the presence of UAC, retry in an UAC compatible manner.
      */
     static int runElevated(File agentExe, String command, TaskListener out, File pwd) throws IOException, InterruptedException {
-        try {
-            return new LocalLauncher(out).launch().cmds(agentExe, command).stdout(out).pwd(pwd).join();
-        } catch (IOException e) {
-            if (e.getMessage().contains("CreateProcess") && e.getMessage().contains("=740")) {
-                // fall through
-            } else {
-                throw e;
-            }
-        }
-
-        // error code 740 is ERROR_ELEVATION_REQUIRED, indicating that
```

Comment:
IIRC this code did not quite work on Windows 10 anyway. Do not recall details. Switching to a built-in elevation logic is a right step anyway